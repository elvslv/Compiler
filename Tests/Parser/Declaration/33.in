type a  = record c, b: a; end;