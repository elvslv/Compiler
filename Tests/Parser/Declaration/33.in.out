24 1 Unknown type